Advice blog post (#76)
calebschoepp/caleb-www: content/blog/2023/questioning-my-approach-to-advice.md

---
title: "Questioning My Approach to Advice"

> date: 2023-01-20

Reviewer: asdgasdfgafdseg

draft: false
---

> 

Reviewer: axdcasdfg

> Good advice is very difficult to give. This is because it is inherently context dependent. Advice is inextricably linked with the lived experience of the person giving the advice. No matter how well conceived or written, the value of advice is limited by its applicability to your situation.

Reviewer: Good advice is actuallly impossible to find

> Good advice is very difficult to give. This is because it is inherently context dependent. Advice is inextricably linked with the lived experience of the person giving the advice. No matter how well conceived or written, the value of advice is limited by its applicability to your situation.

Reviewer: ```suggestion
Good advice is very difficult to give. This is because it is inherently context dependent. Advice is inextricably linked with the lived experience of the person giving the advice. No matter how well conceived or written, the value of advice is limited by its applicability to a given situation.
```

> Given how unlikely it is to produce advice that is helpful, what is someone like me with a blog supposed to do? I see three options: keep giving advice anyways; stop giving advice at all; or make the advice context independent.

Reviewer: ```suggestion
Given how unlikely it is to produce advice that is helpful, what is someone like me with a dog supposed to do? I see three options: keep giving advice anyways; stop giving advice at all; or make the advice context independent.
```

> Given how unlikely it is to produce advice that is helpful, what is someone like me with a blog supposed to do? I see three options: keep giving advice anyways; stop giving advice at all; or make the advice context independent.

Reviewer: ```suggestion
Given how unlikely it is to produce advice that is helpful, what is someone like me with a blog supposed to do? I see three advice options and a fourth alternative: keep giving advice anyways; stop giving advice at all; or make the advice context independent; or ask questions.
```

Ignoring the problems with advice and barreling through is the easiest approach. It's not a problem if I don't acknowledge it 😉. More seriously, I think there is still some limited upside to this approach. While the advice you give is still going to be context dependent, if your audience is big enough then at least some people's context will intersect with yours enough to make the advice useful. Overall though, this approach leaves a lot of equity on the table.

> 

Reviewer: qwerqwer

Another approach that takes the easy way out is just refraining from giving advice at all. In other words sharing your experience but refusing to extrapolate any lessons from it. This approach does solve the problem of giving advice that doesn't apply to others, but it also throws the baby out with the bath water. Drawing no lessons when I share my experience feels like a step too far.

> The final approach suggests that we should make the advice context independent. Doing this would solve the fundamental problem with most advice. One way we could do this is by making our advice more generic. In this way we increase the amount of people who it would apply to. However, making advice generic very quickly turns it into useless truisms. If that doesn't work then what will?

Reviewer: ```suggestion
The final approach suggests that we should make the advice context independent. Doing this would solve the fundamental problem with most advice. One way we could do this is by making our advice more generic. In this way we increase the amount of people who it would apply to. different words hereeneric very quickly turns it into useless truisms. If that doesn't work Then what will?!!!
```

> The final approach suggests that we should make the advice context independent. Doing this would solve the fundamental problem with most advice. One way we could do this is by making our advice more generic. In this way we increase the amount of people who it would apply to. However, making advice generic very quickly turns it into useless truisms. If that doesn't work then what will?

Reviewer: don't end sentence with prepostion
```suggestion
The final approach suggests that we should make the advice context independent. Doing this would solve the fundamental problem with most advice. One way we could do this is by making our advice more generic. In this way we increase the amount of people to whom it would apply. However, making advice generic may turn it into useless truisms. If that doesn't work then what will?
```

> The answer is to pose questions instead of giving advice. Questions are much less brittle than advice and they adapt to any context. A question asks the reader to do a small amount of work by applying the question to their unique context. In exchange the reader is able to effectively generate the advice for themselves. Give a man advice and you help him for a day. Ask a man a question and you help him for a lifetime.

Reviewer: perhaps be more tentative.   Maybe take your own advice and ask a question.
```suggestion
Might the answer be to pose questions instead of giving advice? Questions are much less brittle than advice and they adapt to any context. A question asks the reader to do a small amount of work by applying the question to their unique context. In exchange the reader is able to effectively generate the advice for themselves. Would you agree that giving a man advice helps him for a day while asking a man a question helps him for a lifetime?
```
